app/models/resturant.rb

```diff
@@ -0,0 +1,8 @@
+class Resturant < ApplicationRecord	
+	belongs_to :cuisine
+	validates :name, presence: true
+	validates :cuisine_id, presence: true
+	validates :rating, presence: true
```

Comment:
I later removed this from restaurants and added it to reviews where I also added this validation